### PR TITLE
Configurate linters.

### PR DESCRIPTION
**Pull request by taher-software**
I created the required files for this project. Please check A.html file and the styling provided in A.css file , the content meet the linters rules. I'm just looking forward your feedback before to merge with master branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+This project,  will set up a "Hello world" repository. No complex coding is required for this project. The main  goal here is to master all of the tools and best practices which we  will use  in all Microverse projects.
+
+# Built With
+* html5 & CSS3
+* linters
+* git & github 
+
+# Live Demo
+
```
Comment (DanteAlonsoHT) [inline README.md:11]: - [ ] You can remove this empty section.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+# Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+## Prerequisites: html5 , css3 , git 
+## Setup : clone this repistory using git clone command line .
+
```
Comment (DanteAlonsoHT) [inline README.md:19]: - [ ] [OPTIONAL] For the next time try to add more instructions about how to set up your project in any local environment, check [This Example](https://github.com/DanteAlonsoHT/Leaderboard_API) to let you know what we expect from students 😉

File: txt.tx [hunk 3]
```diff
@@ -1 +0,0 @@
-This is my first project !
```
Comment (DanteAlonsoHT) [inline txt.tx:1]: - [ ] I feel this is your `test.md`, please if it's that case, please re-named to `test.md` and remove it from your Github repository.

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1> Status: Required Changes 🚫 🔴 </h1>

Hi @Taher-web-dev,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Improve the description in your Pull Request, explain what you did, what is the main scope, which files do you changed or added, what you set up in your project in this pull request, and so on.
- [ ] Fix the linters, all your linters should be green and passed successfully.

image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — APPROVED** [review]
## **Hi @Taher-web-dev**

## **Project Approved ✅**



Your project is complete!  There is nothing else to say other than... it's time to merge it 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**